### PR TITLE
Make the anchor tags include the full path

### PR DESCRIPTION
**Pull request by agubler**
## Description

The anchor tags need to include the full path now that we use the base element to control the global app prefix.

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-g9gc1i0hy.now.sh

File: src/blog/BlogPost.tsx [hunk 1]
```diff
@@ -31,13 +34,21 @@ export function formatDate(date: string) {
...
 		excerpt,
 		path
 	});
 
 	if (result) {
+		url &&
+			decorate(
+				result.content,
+				(node: VNode) => {
+					(node.properties as any).href = `${url}${node.properties.href}`;
```
Comment (KaneFreeman) [inline src/blog/BlogPost.tsx]: Won't this mess up absolute links?

Comment (agubler) [reply src/blog/BlogPost.tsx]: Yes I realised that lol good spot.

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-ir42aiw27.now.sh

**Pull request merged**